### PR TITLE
wysiwyg image resize added

### PR DESCRIPTION
**Pull request by mustafa220**

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -10,5 +11,13 @@ module.exports = {
     if (process.env.NODE_ENV === "development") {
       config.output.filename("[name].[hash].js").end();
     }
+  },
+  configureWebpack: {
```
Comment (rijkvanzanten) [inline vue.config.js]: Is there any way we can make this quill plugin work without having to inject Quill in the global scope?

Comment (benhaynes) [reply vue.config.js]: Are we still planning on moving to TinyMCE as our main/only WYSIWYG?

Comment (shealavington) [reply vue.config.js]: Wasnt it a move to TipTap?

Comment (benhaynes) [reply vue.config.js]: Haha, yeah... but more recently @rijkvanzanten has been thinking TinyMCE might be the more flexible.

Comment (rijkvanzanten) [reply vue.config.js]: TipTap has way too much overhead unfortunately. TinyMCE is a little more plug and play and hands off for us

**Comment by rijkvanzanten** [comment]
Closing this in favor of the new TinyMCE based wysiwyg editor

**Pull request closed without merging**